test/integration.js

```diff
@@ -478,20 +476,20 @@ describe('Parquet', function() {
     //   const opts = { useDataPageV2: true, compression: 'LZO' };
     //   return writeTestFile(opts);
     // });
-
+    //
```

Comment:
Could we drop this commented out code while we're at it?

Reply:
Yes, see my comment above.